app/adapters/whats_app_adapter/three_sixty_dialog_outbound.rb

```diff
@@ -114,17 +114,15 @@ def send_message_template(recipient, message)
       def send_message(recipient, message)
         files = message.files
 
-        if files.blank?
-          WhatsAppAdapter::ThreeSixtyDialogOutbound::Text.perform_later(organization_id: message.organization.id,
-                                                                        payload: text_payload(
-                                                                          recipient, message.text
-                                                                        ),
-                                                                        message_id: message.id)
-        else
-          files.each do |_file|
```

Comment:
We used to enqueue `UploadFileJob`s for every `_file`. Does it matter that we enqueue only one job now?

Reply:
This actually was an oversight. We iterated through every file attached to a message to schedule a job to upload the file, but then iterated through a request's files to actually upload them so we were uploading many more files than were necessary.